core/src/main/java/com/graphhopper/routing/util/spatialrules/countries/GermanySpatialRule.java

```diff
@@ -49,10 +50,10 @@ public double getMaxSpeed(String highwayTag, double _default) {
     }
 
     @Override
-    public Access getAccess(String highwayTag, TransportationMode transportationMode, Access _default) {
+    public RoadAccess getAccess(String highwayTag, TransportationMode transportationMode, RoadAccess _default) {
         if (transportationMode == TransportationMode.MOTOR_VEHICLE) {
             if (highwayTag.equals("track"))
-                return Access.CONDITIONAL;
+                return RoadAccess.FORESTRY;
```

Comment:
Isn't this often Agricultural?

[This](https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access-Restrictions#Germany) page proposes destination.

Reply:
I think I was trying to satisfy some unit tests but will try `destination`

Reply:
I think it's worth discussing about this :). Essentially for most cases Agricultural or Forestry will results in no access. Whereas destination would result in access, but with a high weight or an actual destination check. I am not sure which is the better solution.

Reply:
Ok, for Germany I'll do destination only. But for Austria we should keep the access to forestry (general access is `no`). Agriculture shouldn't be used in those cases. At least they say this:

> Note a farmer's access track would be private rather than agricultural unless the track is open to any vehicle used for agricultural purposes. 

https://wiki.openstreetmap.org/wiki/Key:access

Reply:
Please see this commit: https://github.com/graphhopper/graphhopper/commit/4896d85452d68a686da263b08fac9f7010f733da